pkgs/tools/networking/networkmanager_dmenu/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, glib, fetchFromGitHub, networkmanager, python3Packages
+, gobjectIntrospection, dmenu, makeWrapper }:
+
+let inherit (python3Packages) python pygobject3;
+in stdenv.mkDerivation rec {
+  name = "networkmanager_dmenu-${version}";
+  version = "git-20170301";
```

Comment:
Please use `networkmanager_dmenu-uunstable-<commit_date>` (example: `networkmanager_dmenu-unstable-2014-09-23`) for unstable packages here.
(Side note for other maintainers: The package maintainer seems to be not eager to do regularly releases. He maintains a git-based Archlinux package.)